sources/packages/load-test/src/utils/auth/token-helpers.ts

```diff
@@ -51,12 +54,16 @@ function needRenewJwtToken(jwtExp: number, maxSecondsToExpired = 0): boolean {
  */
 export function getCachedToken(
   authorizedParty: AuthorizedParties,
-  credentials: UserPasswordCredential,
+  credentials: AuthenticationCredential,
   options?: { uniqueTokenCache?: string }
 ): string {
-  const tokenCacheKey = `load_test_token_cache_${
-    options?.uniqueTokenCache ?? credentials.userName
-  }`;
+  let credentialTokenKey: string = authorizedParty;
+  if (authorizedParty !== AuthorizedParties.LoadTestGateway) {
```

Comment:
I believe the test should rely on the `credentials` type only. It is not a blocker because we will not probably add many clients.